.cspell.json

```diff
@@ -0,0 +1,174 @@
+      "spacebar",
+      "WCAG",
+      "FOUC",
+      "datepicker",
+      "GCHQ",
+      "desending",
+      "Bloggs",
+      "statustext",
+      "helpertext",
+      "whenup",
+      "doubleclick",
+      "Modernizr",
+      "Macchiato",
+      "xxxs",
+      "Rayes",
+      "testspec",
+      "sonarjs",
+      "arrowup",
+      "arrowdown",
+      "arrowleft",
+      "arrowright",
+      "setinput",
+      "validationstatus",
+      "htmlfor",
+      "popperjs",
+      "doubleespresso",
+      "flatwhite",
+      "cappucino",
+      "Ameno",
+      "Aicano",
+      "Menukey",
+      "hidelabel",
+      "apperance",
+      "disableclick",
+      "disablehover",
+      "validationinlineinternal",
+      "Prehydration",
+      "vdom",
+      "hiddeninputvalue",
+      "selectchange",
+      "componentdidload",
+      "Cancelation",
+      "keydown",
+      "autocorrection",
+      "keyup",
+      "connnector",
+      "mouseleave",
+      "fontsource",
+      "Nunito",
+      "ruleset",
+      "ICDSTEST",
+      "fakepath",
+      "mybuttonid",
+      "viewbox",
+      "interactable",
+      "adipiscing",
+      "elit",
+      "eiusmod",
+      "tempor",
+      "incididunt",
+      "Letraset",
+      "grouplinks",
+      "facilisis",
+      "dignissim",
+      "laoreet",
+      "molestie",
+      "ullamcorper",
+      "Phasellus",
+      "congue",
+      "nisl",
+      "linears",
+      "Pageheader",
+      "Cortado",
+      "Ristretto",
+      "Reallylongonewordanswerthatcouldbeatypeofespresso",
+      "someespresso",
+      "CAPUCCINO",
+      "childgroup",
+      "Antidisestablishmentarianismbutmadeevenlonger",
+      "Liberica",
+      "maxlengthtext",
+      "Applicationwithareallyreallylongsinglewordname",
+      "eleifend",
+      "justo",
+      "vulputate",
+      "accumsan",
+      "rhoncus",
+      "aliquet",
+      "sollicitudin",
+      "porttitor",
+      "ultricies",
+      "augue",
+      "volutpat",
+      "tincidunt",
+      "Praesent",
+      "euismod",
+      "consequat",
+      "feugiat",
+      "nibh",
+      "placerat",
+      "hendrerit",
+      "quxx",
+      "doloe",
+      "nostrud",
+      "ullamco",
+      "aliquip",
+      "testbutton",
+      "galão",
+      "kopi",
+      "luwak",
+      "panna",
+      "affogato",
+      "frappuccino",
+      "mazagran",
+      "nonummy",
+      "Reallylongonewordanswerthatcouldbethenameofacoffee",
+      "visuallyhidden",
+      "myfunc",
+      "someurl",
+      "architechtural",
+      "fullwidth",
+      "arialivemode",
+      "canvastext",
+      "slotchange",
+      "focussable",
+      "keyline",
+      "fullheight",
+      "texy",
+      "scrolltotop",
+      "prepopulate",
+      "searchoption",
+      "icoptionselect",
+      "lobortis",
+      "imperdiet",
+      "sociosqu",
+      "himenaeos",
+      "charcount",
+      "autoprops",
+      "mycontrolname",
+      "icchange",
+      "fadein",
+      "searchbar",
+      "trunc",
+      "merker",
+      "necolas"
```

Comment:
Please remove the following words as they are incorrectly spelt: "desending", "merker", "connnector", "architechtural"

With the use of the lorem ipsum cspell dictionary, can the lorem ipsum words in this list be taken out? Are there other words that can be taken out now that you've installed the relevant dictionary e.g. is htmlFor covered in the HTML dictionary?